tests/test_dt.py

```diff
@@ -166,3 +166,20 @@ def test_total_minutes(timedeltas: timedelta) -> None:
     assert result_pda == result_pl
     assert result_pdn == result_pl
     assert result_pdns == result_pl
+
+
+@pytest.mark.parametrize("constructor", [pd.DataFrame, pl.DataFrame])
+@pytest.mark.parametrize(
+    "fmt", ["%Y-%m-%d", "%Y-%m-%d %H:%M:%S", "%Y/%m/%d %H:%M:%S", "%G-W%V-%u", "%G-W%V"]
+)
+def test_dt_strftime(constructor: Any, fmt: str) -> None:
+    input_frame = nw.from_native(constructor(data))
+    input_series: nw.Series = input_frame["a"]  # type: ignore[assignment]
```

Comment:
If I don't specify this, then line 181 complains that `input_series` may be series _or_ dataframe, and dataframe does not have a `.dt` attribute